devel/unretire-literally-anyone.lua

```diff
@@ -0,0 +1,69 @@
+local dialogs = require 'gui.dialogs'
```

Comment:
The diff between this and the existing `unretire-anyone.lua` script is:
```
55,57c33,41
<       if (histFig.died_year == -1 or histFlags.ghost) and not histFlags.deity and not histFlags.force then
<         local creature = df.creature_raw.find(histFig.race).caste[histFig.caste]
<         local name = creature.caste_name[0]
---
> 
>       local creature = df.creature_raw.find(histFig.race).caste[histFig.caste]
>       local name = creature.caste_name[0]
>         if histFig.died_year >= -1 then
>             histFig.old_year = df.global.cur_year
>             histFig.old_seconds = df.global.cur_year_tick + 1
>             histFig.died_year = -1
>             histFig.died_seconds = -1
>         end
79d62
<     end
```
Why is this a separate script instead of a modification to the existing script?

Reply:
(when you integrate into the existing script, please feel free to remove in the in-script help text. We're trying to clean those up and get rid of them as we do these PRs)

Reply:
Because I'm new here, and wasn't sure what the protocol was. And was slightly uncomfortable with modifying somebody else's script wholesale without getting prior approval from the author.

Reply:
I think that behavior is granted. could you add the functionality to the existing `unretire-anyone` and add a flag to enable it?

Reply:
There's a nice simple example of how to process argument flags in autonick: https://github.com/DFHack/scripts/blob/master/autonick.lua#L32-L38

Reply:
Done.